spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -467,6 +483,11 @@ public void setSize(Integer size) {
 	public static class Listener {
 
 		/**
+		 * Container type.
+		 */
+		private ContainerType type = ContainerType.SIMPLE;
```

Comment:
I don't know all of the intimate details, but the description above for `ContainerType.SIMPLE` is that it is the legacy container type. Wouldn't it be better to default to the non-legacy type with the new major version of Spring Boot (and Spring AMQP)?